code/switch.js

```diff
@@ -0,0 +1,192 @@
+const foodType = prompt(
+  `What do you feel like eating today? Type a number 1-3.
+  \n1. Pizza
+  \n2. Pasta
+  \n3. Salad`
+)
+
+// Chosen food is saved in a variable and then shown to the customer via an alert, using an switch case
+let chosenFood
+let pizzaType
+let pastaType
+let saladType
+let subtypeChoice
+
+// Step 3 - Subtype choice
+// Nested switch case for checking subtype choices at the same time as main choice. 
+switch (foodType) {
+  case "1":
+    chosenFood = "Pizza"
+    alert(
+      `You've chosen ${chosenFood}!`
+    )
+    pizzaType = prompt(
+      `Delicious! What kind of ${chosenFood} do you want? Type a number.
+    \n1. Quattro Formaggio
+    \n2. Margherita
+    \n3. Swedish Kebab`
+    )
+    switch (pizzaType) {
+      case "1":
+        subtypeChoice = "Quattro Formaggio"
+        alert(
+          `You chose a ${subtypeChoice} ${chosenFood}.`
+        )
+        break
+      case "2":
+        subtypeChoice = "Margherita"
+        alert(
+          `You chose a ${subtypeChoice} ${chosenFood}.`
+        )
+        break
+      case "3":
+        subtypeChoice = "Swedish Kebab"
+        alert(
+          `You chose a ${subtypeChoice} ${chosenFood}.`
+        )
+        break
+    }
+    break
+  case "2":
+    chosenFood = "Pasta"
+    alert(
+      `You've chosen ${chosenFood}!`
+    )
+    pastaType = prompt(
+      `Delicious! What kind of ${chosenFood} do you want? Type a number.
+    \n1. Truffle
+    \n2. Bolognese
+    \n3. Carbonara`
+    )
+    switch (pastaType) {
+      case "1":
+        subtypeChoice = "Truffle"
+        alert(`
+        You chose a ${subtypeChoice} ${chosenFood}.`
+        )
+        break
+      case "2":
+        subtypeChoice = "Bolognese"
+        alert(
+          `You chose a ${subtypeChoice} ${chosenFood}.`
+        )
+        break
+      case "3":
+        subtypeChoice = "Carbonara"
+        alert(
+          `You chose a ${subtypeChoice} ${chosenFood}.`
+        )
+        break
+    }
+    break
+  case "3":
+    chosenFood = "Salad"
+    alert(`You've chosen ${chosenFood}!`)
+    saladType = prompt(`Delicious! What kind of ${chosenFood} do you want? Type a number.
+    \n1. Ceasar
+    \n2. Goat Cheese
+    \n3. Vegan`)
+    switch (saladType) {
+      case "1":
+        subtypeChoice = "Ceasar"
+        alert(
+          `You chose a ${subtypeChoice} ${chosenFood}.`
+        )
+        break
+      case "2":
+        subtypeChoice = "Goat Cheese"
+        alert(
+          `You chose a ${subtypeChoice} ${chosenFood}.`
+        )
+        break
+      case "3":
+        subtypeChoice = "Vegan"
+        alert(
+          `You chose a ${subtypeChoice} ${chosenFood}.`
+        )
+        break
+    }
+    break
+  default: alert(
+    `I'm sorry, we don't have that on the menu.`
+  )
+}
+
+// Step 4 - Age
+// Step 5 - Order confirmation
+// I decided to nest the switch cases here aswell, so stept 4-5 are in the same section
```

Comment:
Great idea to put step 4 and 5 together! Looks clean and does not interfere with the understanding of the code.